### DISCUSSION TITLE
在PC部署alist之后，电脑可以正常访问，但是同一局域网的手机和电视都无法访问怎么办？

### DISCUSSION THREAD

**Discussion by zwykoller** [AlistGo/alist#3161] [Q&A]
alist是最新版，用PC浏览器和potplayer都可以正常访问、播放。但是手机和电视都无法通过webdav访问，我试过nplayer，mixplorer，infuse都连接不上，用手机浏览器直接访问PC的IP也连不上，请教各位大佬该怎么办？

**Comment by szhyf** [comment]
防火墙端口没打开吧

**Reply by zwykoller** [reply]
果然是这个，我把防火墙都关了就可以了，谢谢啦，不过如果防火墙打开的时候该怎么设置呢

**Reply by szhyf** [reply]
在防火墙设置里把alist设置成允许应用

**Reply by zwykoller** [reply]
好嘞，感谢帮助：）

**Reply by gl980418** [reply]
我防火墙全关了也访问不了，大佬知道怎么办吗

**Reply by 1273257194** [reply]
同样的问题，也试过防火墙全部关闭。电脑不管是ping127、ping自己的ip、ping手机ip都是通的。但是es文件浏览器、infuse都连接不上alist，在cmd上显示一个条信息后就没有反应了。在es浏览器上显示的错误日志是ping端口不通，显示确认服务器状态。对了手机浏览器输入电脑ip加5244也是没有反应。今天到公司测试手机是可以连接上alist的。请大佬看看这是哪里的问题。

**Reply by explorer-exe-bilibili** [reply]
我这也是，用的是香橙派挂载alist，香橙派上用127.0.0.1:5244可以正常访问，但是在同一局域网内的电脑不能用ip:5244访问alist，ping ip可以ping的通但是ping ip:5244就ping不了了，用浏览器会显示：当前无法使用此页面192.168.1.19 未发送任何数据。ERR_EMPTY_RESPONSE

**Reply by xiaoning356** [reply]
试试把config.json里的address改成默认的0.0.0.0

**Reply by ghost** [reply]
> 试试把config.json里的address改成默认的0.0.0.0

为默认 然而还是无法访问

**Reply by hugh5211** [reply]
电脑先设置下网络共享文档，手机能连上自然也就能连Alist

**Reply by ghost** [reply]
> 差不多同样的问题  手机 电脑都能直接网页访问，但是手机 电脑infuse都挂载不上alist，提示HTTP连接错误在路径加上/dav就可以了。。。。

**Reply by ghost** [reply]
> 电脑先设置下网络共享文档，手机能连上自然也就能连Alist

已经解决。加上去/dav。浏览器访问正常，然后其他软件加上这个没问题了

**Comment by Cooooldwind** [comment]
我这是防火墙关了可以开了就不行

**Reply by ghost** [reply]
我还是直接用服务器好了🤔

**Reply by explorer-exe-bilibili** [reply]
端口5244要打开，打开以后就可以不用关防火墙了

**Reply by lmx4727** [reply]
我把端口打开了，但是其他设备还不能访问

**Comment by githubdulong** [comment]
大概率是你直接用 127 那个 ip 了？
其他设备需要使用挂载的那台电脑本机 ip 替换 127ip 就行了

**Reply by AppleJack-eng** [reply]
还是不行

**Reply by explorer-exe-bilibili** [reply]
也许你可以试试用openp2p把5244端口映射到你用的机器上再用127.0.0.1:5244试试看，如果可以的话大概率就是防火墙或者IP错了的问题了

**Reply by WXqwer** [reply]
成功了！

**Comment by explorer-exe-bilibili** [comment]
> alist是最新版，用PC浏览器和potplayer都可以正常访问、播放。但是手机和电视都无法通过webdav访问，我试过nplayer，mixplorer，infuse都连接不上，用手机浏览器直接访问PC的IP也连不上，请教各位大佬该怎么办？

设置怎么设置的？路径要填/dav

**Reply by kalikkk1234** [reply]
为什么，我这里显示Method Not Allowed

**Comment by kkrootl** [comment]
我是电脑手机网页都可以正常访问 防火墙也设置了    sen  vidhub infuse  都连不上  日志里显示一堆连接 但就是连接失败  这咋整
